src/pages/documentation.md

```diff
@@ -0,0 +1,191 @@
+---
```

Comment:
Weren't we gonna just add a reference to the documentation file on the gh repo?

Reply:
we were but I had issues with the parser so I ended up just copying the file, might look into it in the future